### PR TITLE
Updated OSD branding

### PR DESCRIPTION
**Pull request by jewzaam**
https://issues.redhat.com/browse/OSD-3122

Requires blocking issue https://issues.redhat.com/browse/APPSRE-2631 be fixed.

### CODE REVIEW

**Comment by jewzaam** [comment]
/hold

Blocked by https://issues.redhat.com/browse/APPSRE-2631

**Comment by jewzaam** [comment]
APPSRE-2631 is done!

/hold cancel

**Comment by jewzaam** [comment]
/hold

Err.. we're doing a SSS dance to get another change out, putting on hold until that's completed.
https://issues.redhat.com/browse/OSD-5566

File: README.md [hunk 1]
```diff
@@ -50,12 +50,13 @@ direct:
...
 
 * `matchLabels` (optional, default: `{}`) - adds additional `matchLabels` conditions to the SelectorSyncSet's `clusterDeploymentSelector`
 * `matchExpressions` (optional, default: `[]`) - adds `matchExpressions` conditions to the SelectoSyncSet's `clusterDeploymentSelector`
-* 1resourceApplyMode1 (optional, default: `"Sync"`) - sets the SelectorSyncSet's `resourceApplyMode`
+* `resourceApplyMode` (optional, default: `"Sync"`) - sets the SelectorSyncSet's `resourceApplyMode`
 * `matchLabelsApplyMode` (optional, default: `"AND"`) - When set as `"OR"` generates a separate SSS per `matchLabels` conditions. Default behavior creates a single SSS with all `matchLabels` conditions.  This is to tackle a situation where we want to apply configuration for one of many label conditions.
+* `applyBehavior` (optiona, default: None, [see hive default](https://github.com/openshift/hive/blob/master/config/crds/hive.openshift.io_selectorsyncsets.yaml)) - sets the SelectorSyncSet's `applyBehavior`
```
Comment (cblecker) [inline README.md:59]:
```suggestion
* `applyBehavior` (optional, default: None, [see hive default](https://github.com/openshift/hive/blob/master/config/crds/hive.openshift.io_selectorsyncsets.yaml)) - sets the SelectorSyncSet's `applyBehavior`
```

**Comment by jewzaam** [comment]
Technically not done w/ OSD-5566 but it's been hitting issues and is reverted to a state that's OK to try out this PR again.

**Pull request merged**